scripts/osx-create-dmg.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+APP="frontends/qt/build/osx/BitBox.app"
+INSTALLER="frontends/qt/build/osx/BitBoxApp Installer.dmg"
```

Comment:
Please remove the space in the filename as spaces are a bit annoying to deal with on the terminal. Before release the file will be manually renamed containing the version anyway.